wordpress-mobile/WordPress-Android
# [Aztec] Fix an issue where changes made to the HTML editor are not posted on the site

This PR fixes an issue where changes made to a post in the HTML editor are not posted to the site.

Tests 1:
- Edit a post
- On Aztec switch to HTML without changing the post in Visual Editor
- Make same changes
- Tap Update
(the app should post the new content)

Tests 2:
- Edit a post
- On Aztec switch to HTML without changing the post in Visual Editor
- Do NOT make changes
- Tap Update
(the app should NOT post the new content)

Tests 3:
- Edit a post
- On Aztec switch to HTML without changing the post in Visual Editor
- Make some changes
- Switch back to Visual Editor and Do NOT make changes
- Tap Update
(the app should post the new content on the site)


Note: This should be merged after https://github.com/wordpress-mobile/AztecEditor-Android/pull/682
